jetty-websocket/javax-websocket-tests/src/test/java/org/eclipse/jetty/websocket/javax/tests/server/LargeContainerTest.java

```diff
@@ -94,6 +94,10 @@ public void testEcho() throws Exception
             URI uri = wsb.getWsUri();
 
             WebAppContext webapp = wsb.createWebAppContext();
+            // TODO It's not great that we have to expose these APIs for testing as it can hide other classpath issues!
+            webapp.getSystemClassMatcher().include("org.eclipse.jetty.websocket.javax.");
+            webapp.getServerClassMatcher().exclude("org.eclipse.jetty.websocket.javax.");
```

Comment:
This is red flag for me.
It shouldn't be a requirement to manually manipulate the classloader just to use javax.websocket.
Why isn't the SCI doing this (either normally as a SCI, or via a manual call to it's `.configure()` method)?

Reply:
@joakime I don't think you are understanding the problem - this is not necessary for general embedded usage.  It is only necessary for our own test classes, because we have written some test endpoints that live in `org.eclipse.jetty...` which is the same package that contains our implementation... which is hidden from the webapp.

If our test endpoints were in `org.example`, it would not be a problem.

Reply:
I thought I understood how the hidden classes worked.
But I'm still not sure why this is suddenly an issue, when it wasn't in the past.
I guess I'm more asking why did this change?
What other change caused this?

Also, If we move the endpoints around, we need to be careful of the links to example code from the documentation into our test cases.
But that's a minor niggle, one that's easy enough to correct as you go. (but is often overlooked)

Reply:
there were no asciidoctor warnings when building so I don't think these files were referenced in documentation

Reply:
@joakime it wasn't an issue in the past because we exposed our websocket implementation in the classpath before - so the test classes we created there were also visible.

Now we are hiding our implementation (as we should do), so the test classes we created there are also hidden.

Reply:
Cool, then i'm good with the current state of the PR.

Should we have used `org.eclipse.jetty.test` instead of `com.acme`? (like issue #3318 points out)